net.solarnetwork.node.weather.ibm.wc/src/net/solarnetwork/node/weather/ibm/wc/WCDayDatumDataSource.properties

```diff
@@ -0,0 +1,20 @@
+triggerCronExpression.key = Schedule
+triggerCronExpression.desc = A \
+	<a href="{0}">\
+	cron expression</a> representing the schedule to sample data at.
+	
+uid.key = Service Name
+uid.desc = A unique name to identify this service with.
+
+apiKey.key = API Key
+apiKey.desc = Your IBM Weather Channel API key.
+
+datumPeriod.key = Datum Period
+datumPeriod.desc = The identifier for the datum period (ie. 6hour, 1day, 7day, 10day)
```

Comment:
If doing the drop-down menu setting, the localized messages could be pulled from this resource via the `messageSource` injected at runtime, or just hard-coded to start.